infrastructure/src/main/java/org/corfudb/infrastructure/ManagementServer.java

```diff
@@ -411,23 +399,19 @@ public void handleHealingDetectedMsg(CorfuPayloadMsg<DetectorMsg> msg,
      * Default NodeState has been providing unless the node is not bootstrapped.
      * Failure detector updates ClusterNodeState by current state then current NodeState can be provided to other nodes.
      *
-     * @param msg corfu message containing NODE_STATE_REQUEST
-     * @param ctx netty ChannelHandlerContext
-     * @param r server router
+     * @param req  a message containing a QUERY_NODE request
+     * @param ctx  the netty ChannelHandlerContext
+     * @param r    the server router
      */
-    @ServerHandler(type = CorfuMsgType.NODE_STATE_REQUEST)
-    public void handleNodeStateRequest(CorfuMsg msg, ChannelHandlerContext ctx, IServerRouter r) {
+    @RequestHandler(type = PayloadCase.QUERY_NODE_REQUEST)
+    public void handleQueryNode(RequestMsg req, ChannelHandlerContext ctx, IServerRouter r) {
         NodeState nodeState = clusterContext.getClusterView()
                 .getNode(serverContext.getLocalEndpoint())
                 .orElseGet(this::buildDefaultNodeState);
 
-        r.sendResponse(ctx, msg, CorfuMsgType.NODE_STATE_RESPONSE.payloadMsg(nodeState));
-    }
-
-    @ServerHandler(type = CorfuMsgType.FAILURE_DETECTOR_METRICS_REQUEST)
-    public void handleFailureDetectorMetricsRequest(CorfuMsg msg, ChannelHandlerContext ctx, IServerRouter r) {
-        FailureDetectorMetrics metrics = serverContext.getFailureDetectorMetrics();
-        r.sendResponse(ctx, msg, CorfuMsgType.FAILURE_DETECTOR_METRICS_RESPONSE.payloadMsg(metrics));
```

Comment:
Why is this missing from the PR?

Reply:
It isn't currently being used anywhere and hasn't been active in a year. I spoke with @xnull and it seems fine to remove.